I had commented out much of the adb device interaction code for local testing. 

It seems mozdevice isn't compatible with python 3.

from the test run:

```
[task 2019-11-15T19:41:37.446Z] Traceback (most recent call last):
[task 2019-11-15T19:41:37.447Z]   File "/builds/taskcluster/script.py", line 245, in <module>
[task 2019-11-15T19:41:37.447Z]     sys.exit(main())
[task 2019-11-15T19:41:37.447Z]   File "/builds/taskcluster/script.py", line 157, in main
[task 2019-11-15T19:41:37.447Z]     devices = adbhost.devices()
[task 2019-11-15T19:41:37.447Z]   File "/usr/local/lib/python3.6/dist-packages/mozdevice/adb.py", line 509, in devices
[task 2019-11-15T19:41:37.447Z]     match = re_device_info.match(line)
[task 2019-11-15T19:41:37.447Z] TypeError: cannot use a string pattern on a bytes-like object
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,10 +1,11 @@
-FROM ubuntu:18.04
+FROM ubuntu:18.04@sha256:017eef0b616011647b269b5c65826e2e2ebddbe5d1f8c1e56b3599fb14fabec8
 
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
     apt-get install -y \
     curl \
+    vim \
     dnsutils \
     ffmpeg \
     gettext-base \

--- a/taskcluster/script.py
+++ b/taskcluster/script.py
@@ -47,6 +47,8 @@ def get_device_type(device):
         pass
     elif device_type == "Moto G (5)":
         pass
+    elif device_type == "Android SDK built for x86":
+        pass
     else:
         fatal("Unknown device ('%s')! Contact Android Relops immediately." % device_type, retry=False)
     return device_type
@@ -82,6 +84,8 @@ def enable_charging(device, device_type):
                 device.shell_bool(
                     "echo %s > %s" % (1, g5_path), root=True, timeout=ADB_COMMAND_TIMEOUT
                 )
+        elif device_type == "Android SDK built for x86":
+            pass
         else:
             fatal("Unknown device ('%s')! Contact Android Relops immediately." % device_type, retry=False)
     except ADBError as e:

--- a/taskcluster/script.py
+++ b/taskcluster/script.py
@@ -217,6 +217,7 @@ def main():
         rc = proc.poll()
         if line:
             bytes_written += sys.stdout.write(str(line.decode()))
+        # print("ll: %s, bw: %s, br: %s, rc: %s" % (line_len, bytes_written, bytes_read, rc))
         if line_len == 0 and bytes_written == bytes_read and rc is not None:
             break
     print("script.py: command finished (bytes read: %s, bytes written: %s)" % (bytes_read, bytes_written))

